paasta_tools/cli/cmds/local_run.py

```diff
@@ -352,13 +352,14 @@ def get_container_name():
     return 'paasta_local_run_%s_%s' % (get_username(), randint(1, 999999))
 
 
-def get_docker_run_cmd(memory, random_port, container_name, volumes, env, interactive, docker_hash, command):
+def get_docker_run_cmd(memory, random_port, container_name, volumes, env, interactive, docker_hash, command, hostname):
     cmd = ['docker', 'run']
     for k, v in env.iteritems():
         cmd.append('--env=\"%s=%s\"' % (k, v))
     # We inject an invalid port as the PORT variable, as marathon injects the externally
```

Comment:
This comment is now kinda on the wrong line, try to keep this comment and the --env=port together